Makefile

```diff
@@ -32,6 +36,9 @@ format:
 generate:
 	go generate ./...
 
+install-go-test-coverage:
+	@go install github.com/vladopajic/go-test-coverage/v2@latest
```

Comment:
What is the meaning of `@`?

Reply:
Use `@` will not print execution messages. I'll delete `@`.